sphinxcontrib/jupyter/writers/make_pdf.py

```diff
@@ -25,12 +25,14 @@ class MakePDFWriter():
     def __init__(self, builder):
         self.pdfdir = builder.outdir + "/pdf" #pdf directory 
         self.texdir = builder.outdir + "/executed" #latex directory 
+        self.texbookdir = builder.outdir + "/texbook" # tex files for book pdf
 
         for path in [self.pdfdir, self.texdir]:
             ensuredir(path)
 
         self.pdf_exporter = PDFExporter()
         self.tex_exporter = LatexExporter()
+        self.index_book = builder.config['jupyter_pdf_book_index']
```

Comment:
@AakashGfude let's leave this for now - but it is confusing that this configuration has a different name